pennylane/vqe/vqe.py

```diff
@@ -76,6 +77,7 @@ def __init__(self, coeffs, observables):
 
         self._coeffs = coeffs
         self._ops = observables
+        self.simplify()
```

Comment:
Question: do we want to add `simplify=True` to the signature, and then have
```python
if simplify:
    self.simplify()
```
This way, if passing a large Hamiltonian, you can turn off the simplification if (a) you know it is not needed, and (b) it is too time consuming.

Another question: should the default be true or false?

Reply:
@josh146 I just made this change, and set the default value of `simplify` to `True`. I like this option much better!

Reply:
😆 Yes, I imagine lots of tests are assuming the Hamiltonian structure stays exactly as it was created!